ui/src/components/loading.tsx

```diff
@@ -9,21 +9,27 @@ import React, { ReactNode, useEffect, useRef, useState } from "react";
 type Status = {
   error?: Error;
   isLoading?: boolean;
+  isMutating?: boolean;
   mutate?: () => void;
 };
 
 type Props = {
   size?: string;
   status?: Status;
   children?: React.ReactNode;
+  showProgressOnMutate?: boolean;
```

Comment:
Does progress here indicate the number of retries (e.g. 4 out of max 10 retries) or the number of calls to the export endpoint (e.g. 2 out of 5 signed urls generated), or something else?

Reply:
No, it's just the generic progress spinner that's shown while loading initially. In other cases, backend updates or retries like changing a cohort are done in the background but for export we explicitly want to show the request is in progress because the user is waiting for it.